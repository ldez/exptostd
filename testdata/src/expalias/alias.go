package expalias

import (
	aliased "golang.org/x/exp/maps"
	"golang.org/x/exp/slices" // want "Import statement can drop `golang.org/x/exp` prefix"
)

func _(m map[string]string, a, b []string) {
	aliased.Clone(m)
	slices.Equal(a, b) // want `golang.org/x/exp/slices\.Equal\(\) can be replaced by slices\.Equal\(\)`
}
