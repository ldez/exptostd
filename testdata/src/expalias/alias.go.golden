package expalias

import (
	"golang.org/x/exp/maps"
	aliased "golang.org/x/exp/maps"
	"slices" // want "Import statement 'golang.org/x/exp/slices' can be replaced by 'slices'"
)

func _(m map[string]string, a, b []string) {
	aliased.Clone(m)
	maps.Clone(m)      // want `golang.org/x/exp/maps\.Clone\(\) can be replaced by maps\.Clone\(\)`
	slices.Equal(a, b) // want `golang.org/x/exp/slices\.Equal\(\) can be replaced by slices\.Equal\(\)`
}
